exonum/src/events/mod.rs

```diff
@@ -213,6 +213,6 @@ where
 }
 
 
-fn tobox<F: Future + 'static>(f: F) -> Box<Future<Item = (), Error = F::Error>> {
+fn to_box<F: Future + 'static>(f: F) -> Box<Future<Item = (), Error = F::Error>> {
```

Comment:
Are you sure that this one needs to be renamed?

Reply:
I disliked `tobox` too, but didn't want to "nitpick", so I'm happy now. :smile:

Reply:
Pretty sure, yes. Now it uses common naming, as stated in Rust API Guidelines.